src/project-css/database/migrations/2020_03_18_162037_add_iscms_to_collections.php

```diff
@@ -27,6 +25,9 @@ public function up()
      */
     public function down()
```

Comment:
Needs Proper Documentation: 
Use full PHP Documentation Standards.
Be clear on what the parameters of the function are as well as what the return values are supposed to be.